force-app/main/default/lwc/givingSummaryForDetailPage/givingSummaryForDetailPage.js

```diff
@@ -1,7 +1,17 @@
 import { api, LightningElement } from 'lwc';
+import getContactIdByUserId from "@salesforce/apex/DonationHistoryController.getContactIdByUserId";
 
 export default class GivingSummaryDetailPage extends LightningElement {
```

Comment:
The class name here does not line up with the name of the actual LWC. Can we change the class name to match the file/folder name of GivingSummaryForDetailPage?